src/range/splitting.rs

```diff
@@ -38,20 +36,23 @@ impl Serializable for RangeProofSplitting {
     fn serialize(&self) -> Vec<u8> {
         let mut result = Vec::new();
         // append aggregated proofs to the result
-        result.append(&mut usize_to_bytes(
-            self.get_aggregated().len(),
-            AGGREGATED_NUM_BYTE_NUM,
-        ));
+        let len=self.get_aggregated().len() as u16;
```

Comment:
...but changing `AGGREGATED_NUM_BYTE_NUM` check to u16 cast is not the same. Rust will allow an overflow i.e.
```rust
let a: usize = 2_usize.pow(2*8); // 65536
let b = a as u16; // 0
```
For now let's keep the panic instead of allowing overflow, and we can adjust the code later to remove the panic by changing the input type from usize to something else.